cli/tests/testdata/run/permission_args.out

```diff
@@ -1,2 +1,7 @@
-Permission flags have likely been incorrectly set after the script argument. To grant permissions, define them before the script argument.
+
```

Comment:
We should remove the blank line here in the output.

Reply:
@iuioiua I meant this blank line here at the start, not the other blank line. That one was good.

Reply:
Oh, also we should remove the other blank line at the end of the warning. We don't normally surround warning with blank lines.

Reply:
Yup, we don't normally surround warnings with blank lines and the warning is in yellow so it should be differentiated by colour.

Reply:
Sorry, I just realized this is not yellow so I pushed a change that makes it yellow to save you the hassle.